src/mca/rmaps/rank_file/rmaps_rank_file.c

```diff
@@ -88,9 +88,10 @@ static int prte_rmaps_rf_map(prte_job_t *jdata,
     int rc;
     prte_proc_t *proc;
     pmix_mca_base_component_t *c = &prte_mca_rmaps_rank_file_component.super;
-    char *slots;
+    char *slots = NULL;
```

Comment:
Sigh - now you are going to force me to remember why "slots" no longer needed to be preset to NULL 😄 I know there was a reason, not that it particularly matters.

Reply:
Before this commit the variable is set later, but never used. So effectively a dead variable. I've restored the binding functionality associated with it, so I'll need it to be NULL so I can check to see if a binding was expressed by the user either in the rankfile or elsewhere.